build_keyboard.mk

```diff
@@ -280,6 +280,23 @@ ifneq ("$(wildcard $(KEYBOARD_PATH_1)/config.h)","")
     CONFIG_H += $(KEYBOARD_PATH_1)/config.h
 endif
 
+POST_CONFIG_H :=
```

Comment:
Is this required to work properly? 

Eg, can the post_config stuff be moved so this isn't needed?

Reply:
This is necessary.
It is necessary to prevent unnecessary dependencies between header files between modules in lower layers.

For example, quantum/split_common/transport.c needs to define SERIAL_USE_MULTI_TRANSACTION of serial when using RGBLIGHT_SPLIT.
It is not good to solve this with serial.h and rgblight.h.